keyboards/handwired/dactyl_minidox/keyboard.json

```diff
@@ -10,19 +10,56 @@
     },
     "features": {
         "bootmagic": true,
-        "console": false,
-        "command": false,
         "extrakey": true,
         "mousekey": true,
-        "nkro": true
+        "nkro": true,
+        "rgb_matrix": true
     },
     "ws2812": {
         "pin": "D3"
     },
     "rgb_matrix": {
         "center_point": [133, 54],
         "driver": "ws2812",
-        "split_count": [18, 18]
+        "split_count": [18, 18],
+        "layout": [
```

Comment:
LED layout needs adjusting.
![image](https://github.com/user-attachments/assets/4c2607d0-4e0a-48ae-a6da-ca59750ad833)
Might be better to just leave it as is since I'm going to be looking through all of them anyway.